tutorials/conf.py

```diff
@@ -96,7 +96,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'astropy_favicon.ico'
+html_favicon = "_static/astropy_favicon.ico"
```

Comment:
Can you rebase to remove these other commits?

Reply:
Yup, done.